modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowOutlet.tsx

```diff
@@ -127,6 +128,15 @@ const TestflowOutlet = () => {
 		return null;
 	}
 
+	if (testrayTask.dueStatus.key === TaskStatuses.PROCESSING) {
```

Comment:
This should be PROCESSING or OPEN

I can try to forward this and then you send a follow up, if CI fail, you ammend this change